ampd/Dockerfile

```diff
@@ -4,7 +4,6 @@ RUN apt-get update && apt-get install -y clang protobuf-compiler cmake
 WORKDIR /ampd
 COPY ./Cargo.toml ./Cargo.toml
 COPY ./ampd/Cargo.toml ./ampd/Cargo.toml
-COPY ./ampd/Cargo.lock ./ampd/Cargo.lock
```

Comment:
Should also copy the workspace's Cargo.lock file?

Reply:
you mean the one in the main directory? Makes sense